src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import ThemedImage from "@theme/ThemedImage";
+import styles from "./styles.module.css";
+
+/* data of supporters */
+const financial = [];
+
+const infrastructure = [
+  {
+    name: "Anaconda",
+    link: "https://www.anaconda.com/",
+    light: "img/supporters/anaconda_light.svg",
+    dark: "img/supporters/anaconda_dark.svg",
+    width: 250,
+  },
+  {
+    name: "Netlify",
+    link: "https://www.netlify.com/",
+    light: "img/supporters/netlify_light.svg",
+    dark: "img/supporters/netlify_dark.svg",
+    width: 250,
+  },
+  {
+    name: "Zulip",
+    link: "https://zulip.com",
+    light: "img/supporters/zulip.svg",
+    dark: "img/supporters/zulip.svg",
+    width: 250,
+  },
+];
+
+const developer = [
+  {
+    name: "Anaconda",
+    link: "https://www.anaconda.com/",
+    light: "img/supporters/anaconda_light.svg",
+    dark: "img/supporters/anaconda_dark.svg",
+    width: 250,
+  },
+  // {
+  //   name: "IOOS Integrated Ocean Observing System",
+  //   link: "https://ioos.noaa.gov/",
+  //   light: "img/supporters/ioos.png",
+  //   dark: "img/supporters/ioos.png",
+  //   width: 150,
+  // },
+  {
+    name: "NVIDIA",
+    link: "https://www.nvidia.com/",
+    light: "img/supporters/nvidia_light.svg",
+    dark: "img/supporters/nvidia_dark.svg",
+    width: 250,
+  },
+  // {
+  //   name: "Voltron Data",
+  //   link: "https://voltrondata.com/",
+  //   light: "img/supporters/voltron_light.svg",
+  //   dark: "img/supporters/voltron_dark.svg",
+  //   width: 250,
+  // },
```

Comment:
@mariusvniekerk is it okay to also mention y'all here? If that's the case, could you also add that info to [`steering.csv`](https://github.com/conda/governance/blob/7ee27ebfecc3b36f52d926ee4d36eec30e457439/steering.csv#L4)? Thanks!